apps/menu-bar/src/components/Button.tsx

```diff
@@ -48,7 +48,7 @@ export function getStylesForColor(color: Color, theme: ReturnType<typeof useCurr
           theme === 'light'
             ? addOpacity(lightTheme.background.default, 0.8)
             : addOpacity(darkTheme.background.default, 0.4),
-        borderColor: theme === 'light' ? addOpacity(lightTheme.border.default, 1) : 'transparent',
+        borderColor: theme === 'light' ? lightTheme.border.default : darkTheme.border.default,
```

Comment:
I general, it would be nice to have those borders, or even switch to native macOS buttons instead, but I agree that in the devices list case it does not look good. I have removed the border in launch button case and stick to border included by default in other cases. Hope this sounds good. 🙂

Reply:
Yeah I agree, that looks good